tutorial/README.md

```diff
@@ -266,12 +264,46 @@ docker-compose -f docker-compose-oracle.yaml up --build
 cat debezium-with-oracle-jdbc/init/inventory.sql | docker exec -i dbz_oracle sqlplus debezium/dbz@//localhost:1521/ORCLPDB1
 ```
 
-Adjust the host name of the database server and the name of the XStream outbound server in `register-oracle.json` as per your environment.
+The Oracle connector can be used to interact with Oracle either using the Oracle XStreams API or LogMiner.
+
+### XStreams
+
+The connector by default will chose to use the XStreams API, which requires a license for the Golden Gate product
+(which itself is not required be installed, though).
```

Comment:
Good catch. Logged https://issues.redhat.com/browse/DBZ-3241 for changing it to LogMiner by default.